Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -492,9 +511,14 @@ class LoginDataSource: NSObject, UITableViewDataSource {
             self.count = logins.count
             self.titles = titles
             self.sections = sections
+            self.currentQuery = currentQuery
         }
     }
 
+    func getLogins() -> [Login] {
+        return sections.flatMap { $0.value }
```

Comment:
This is a pretty expensive way of doing this: we take the existing sectioned items, build a new array, filter it, then pass it right back in and compute all the sections again. Can we do better?